sklearn/cross_decomposition/_pls.py

```diff
@@ -403,7 +409,8 @@ def inverse_transform(self, X):
 
         Returns
         -------
-        x_reconstructed : array-like of shape (n_samples, n_features)
+        self : object
```

Comment:
This is not an object as well. It should be an `ndarray of shape (n_samples, n_features)`

Reply:
Thanks!

Reply:
```suggestion
        self : ndarray of shape (n_samples, n_features)
```